src/protocols/bdx/TransferFacilitator.h

```diff
@@ -47,6 +47,9 @@ class TransferFacilitator : public Messaging::ExchangeDelegate, public Messaging
     TransferFacilitator() : mExchangeCtx(nullptr), mSystemLayer(nullptr), mPollFreq(kDefaultPollFreq) {}
     ~TransferFacilitator() override = default;
 
+    CHIP_ERROR OnMessageReceived(chip::Messaging::ExchangeContext * ec, const chip::PayloadHeader & payloadHeader,
```

Comment:
Why is this public?  Shouldn't it be protected?

Reply:
done.

Reply:
done.